docs/user/cropping-and-transforming.md

```diff
@@ -191,6 +177,32 @@ writer.add_page(page)
 writer.write("out-pg-transform.pdf")
 ```
 
+### Scaling the page only
+
+To scale the page by `sx` in the X direction and `sy` in the Y direction: 
+
+```python
+from pypdf.generic import RectangleObject
+
+mb = page.mediabox
+
+page.mediabox = self.mediabox.scale(sx, sy)
+```
+
+If you wish to have more control, you can adjust the various page boxes directly:
```

Comment:
This section about manually setting boxes is a straight move from above to put it into a relevant section.

I don't think it actually adds much to the contents of this section, so can remove it if that is preferred?

Reply:
It should not really hurt to keep it, thus I am okay with the current approach here.